rewriter/tests/omit_wrappers/library.h

```diff
@@ -1,8 +1,3 @@
-/*
-RUN: cp %s %t.h
-RUN: ia2-rewriter --omit-wrappers %t.c %t.h -- -I%resource_dir
-RUN: sh -c "[ ! -e %t.c.args ]"
```

Comment:
Not sure what this is doing. Is the Criterion test sufficient to test what this is testing?

Reply:
Sorry that's my bad, this test should've been removed when we switched over to the source rewriter. The header rewriter required more flags to get it working on real codebases and this was testing one of those flags.